packages/controller/lib/zipFiles.js

```diff
@@ -312,13 +296,13 @@ async function _writeOneObject(objects, zip, rootId, filename, options, callback
             try {
                 data.data = JSON.parse(data.data);
             } catch (e) {
-                callback('Cannot parse "' + data.id + '": ' + e);
+                callback(`Cannot parse "${data.id}": ${e}`);
```

Comment:
just because we touch it right now, I would also change it to `e.message`

Reply:
Done